trade-finance-manager-ui/server/middleware/validatePostAddPaymentRequestBody/index.ts

```diff
@@ -12,19 +12,19 @@ import { PremiumPaymentsTableCheckboxId } from '../../types/premium-payments-tab
 
 const isRequestBodyAnObject = (body: unknown): body is object => !body || typeof body === 'object';
 
-const FACILITY_ID_QUERY_REGEX = /facilityIdQuery=(?<facilityIdQuery>\d{4,10})/;
+const PREMIUM_PAYMENTS_FACILITY_ID_QUERY_REGEX = /premiumPaymentsFacilityId=(?<premiumPaymentsFacilityId>\d{4,10})/;
 
-const getFacilityIdQueryFromReferer = (req: Request): string | undefined => {
+const getPremiumPaymentsFacilityIdQueryFromReferer = (req: Request): string | undefined => {
```

Comment:
documentation in this file please so it is easier to understand what is happening.  For the REGEX and functions